jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/explorer/CodeWhispererExplorerActionManager.kt

```diff
@@ -201,7 +207,3 @@ interface CodeWhispererActivationChangedListener {
 fun isCodeWhispererEnabled(project: Project) = with(CodeWhispererExplorerActionManager.getInstance()) {
     checkActiveCodeWhispererConnectionType(project) != CodeWhispererLoginType.Logout
 }
-
-fun isCodeWhispererExpired(project: Project) = with(CodeWhispererExplorerActionManager.getInstance()) {
-    checkActiveCodeWhispererConnectionType(project) == CodeWhispererLoginType.Expired
-}
```

Comment:
remove since it has nuance that if "logout" is expired or not. use `checkActiveCodeWhispererConnectionType` instead and check if the type is `Expired`

Reply:
instead of removing it, can we just add a comment here to explain the nuance? I see duplicated code otherwise. Or rename it to something like "isCodeWhispererRefreshNeeded"